### ISSUE TITLE
Compose button not focusable by keyboard

### ISSUE THREAD

**Issue by dankeck** [thunderbird/thunderbird-android#8230]
### Checklist

- [X] I have used the search function to see if someone else has already submitted the same bug report.
- [X] I will describe the problem with as much detail as possible.

### App version

8.0b1

### Where did you get the app from?

Google Play

### Android version

14

### Device model

Pixel 5

### Steps to reproduce

1. Connect a wireless keyboard to Android device.
2. Open Thunderbird Beta.
3. While viewing the Inbox:
- Hit the tab key on the keyboard multiple times.
- Hit ctrl + tab multiple times.

### Expected behavior

Eventually, keyboard focus will reach the floating compose button using either tab or ctrl + tab.

### Actual behavior

Keyboard focus never reaches the floating compose button.

**Comment by dankeck** [comment]
This is similar to [issue #5852](https://github.com/thunderbird/thunderbird-android/issues/5852) but for a different element.

**Comment by asoucar** [comment]
Taking this on
